lib/cogctl/actions/users.ex

```diff
@@ -9,21 +9,21 @@ defmodule Cogctl.Actions.Users do
   def run(_options, _args, _config, endpoint),
     do: with_authentication(endpoint, &do_list/1)
 
-  defp generate_table_row(username, nil, nil), do: [username, ""]
-  defp generate_table_row(username, first, nil), do: [username, first]
-  defp generate_table_row(username, nil, last), do: [username, last]
-  defp generate_table_row(username, first, last), do: [username, first <> " " <> last]
+  defp generate_table_row(username, nil, nil, email), do: [username, "", email]
+  defp generate_table_row(username, first, nil, email), do: [username, first, email]
+  defp generate_table_row(username, nil, last, email), do: [username, last, email]
+  defp generate_table_row(username, first, last, email), do: [username, first <> " " <> last, email]
```

Comment:
Might be good to break the name creation out into a separate function that just takes `first` and `last`. That's actually what most of this `generate_table_row` function is doing; breaking it apart makes it easier to see what's going on.